xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -71,6 +71,9 @@ bool CPVRChannelGroups::Update(const CPVRChannelGroup &group, bool bUpdateFromCl
     if (group.GroupID() > 0)
       updateGroup = GetById(group.GroupID());
 
+    if (!updateGroup && group.IsInternalGroup() && (updateGroup = GetGroupAll()) && bUpdateFromClient)
```

Comment:
we never get the internal group from the client (only from the database) so this will never be the case.